<launch>
    <param name="use_sim_time" value="true"/>
    <node name="ares_slam_node" pkg="ares_slam" type="ares_slam_node" output="screen">
        <rosparam file="$(find ares_slam)/param/param.yaml" command="load" />
        <param name="base_frame"        value="base_link"/>
        <param name="map_frame"         value="map"/>
        <param name="scan_topic"        value="scan"/>
    </node>
</launch>
